README.md

```diff
@@ -1 +1,131 @@
 Check: [Amending an versioning of dictionaries](https://lifecycle-project.github.io/ds-upload/articles/dictionaryVersioning.html)
+
+# Create a dataset from existing dictionary
+
+The transformation (or creating a dataset from existing dictionary) is performed once and is described here only to explain the process.
+
+The original dictionaries are located here: `lifecycle-project/ds-dictionaries/dictionaries/`.
+
+These dictionaries are saved in an Excel file with two sheets *Variables* and *Categories* and contain the following columns:
+
+*Variables*
+
+-   name
+
+-   valueType
+
+-   unit
+
+-   label
+
+*Categories*
+
+-   variable
+
+-   name
+
+-   isMissing
+
+-   label
```

Comment:
These are list so make it markdown
```markdown
Variables
- name
- valueType
- unit
- label

Categories
- variable
- name
- isMissing
- label
```

Reply:
R markdown does not like this format, so 'old' ugly way is kept